.gitignore

```diff
@@ -14,7 +14,7 @@ out/
 
 # build tool
 .gradle/
-**/build/*
+build/
```

Comment:
Will this still ignore the build directories in alert-common and alert-database?